examples/temperature-measurement-app/esp32/main/include/diagnostic-logs-provider-delegate-impl.h

```diff
@@ -54,12 +55,59 @@ class LogProvider : public DiagnosticLogsProviderDelegate
     LogProvider(const LogProvider &)             = delete;
     LogProvider & operator=(const LogProvider &) = delete;
 
-    // This tracks the ByteSpan for each session
-    std::map<LogSessionHandle, ByteSpan *> mSessionSpanMap;
+    struct CrashLogContext
+    {
+        spi_flash_mmap_handle_t mappedHandle;
+        const void * mappedAddress;
```

Comment:
please init these values in struct

Reply:
I had them, but after adding Reset() I though that would be redundant. Adding them back.

Reply:
yep, usually we prefer init them in members directly. thanks